setup.py

```diff
@@ -23,12 +25,28 @@
         "click",
         "numpy",
         "prettytable",
+        "flake8",
+        "bandit",
+        "isort",
+        "mypy",
+        "pytest",
+        "pytest-cov",
+        "pytest-mock"
```

Comment:
these are test dependencies. No need to add them under `install_requires`

Reply:
i put them in because circleci was failing because it couldnt find them?

Reply:
Figured it out.